Code/Legacy/CrySystem/LevelSystem/SpawnableLevelSystem.cpp

```diff
@@ -536,9 +536,6 @@ namespace LegacyLevelSystem
 
         const AZ::TimeMs beginTimeMs = AZ::GetRealElapsedTimeMs();
 
-        // Clear level entities and prefab instances.
-        EBUS_EVENT(AzFramework::GameEntityContextRequestBus, ResetGameContext);
```

Comment:
Good catch. This change makes sense to me but I'd like to gather some more information to avoid possible bugs.

ResetGameContext() also calls CreateRootSlice() down the line. I think this is required so that further level loads can use this root slice. Since you said that it fixes the https://github.com/o3de/o3de/issues/8696 bug, does that mean that when the second level is loaded, a new root slice is created or the existing one is used?

Reply:
> Good catch. This change makes sense to me but I'd like to gather some more information to avoid possible bugs.
> 
> ResetGameContext() also calls CreateRootSlice() down the line. I think this is required so that further level loads can use this root slice. Since you said that it fixes the #8696 bug, does that mean that when the second level is loaded, a new root slice is created or the existing one is used?

When the second level is loaded, the existing root slice is used. A new one is not created, however the number of entities on the existing root slice have been cleared to 0.